packages/vue/src/components/organisms/SfHero/SfHero.html

```diff
@@ -1,53 +1,50 @@
 <div class="sf-hero">
-    <div class="glide" ref="glide">
-        <div class="glide__track" data-glide-el="track">
-            <ul class="glide__slides sf-hero__slides">
-                <div v-for="item in items">
-                    <SfHeroItem
-                        :background="item.background"
-                        :image="item.image"
-                        :title="item.title"
-                        :subtitle="item.subtitle"
-                        :buttonText="item.buttonText">
-                    </SfHeroItem>
-                </div>
-                <!--@slot default slot for SfHeroItems tags -->
-                <slot />
-            </ul>
+  <div class="glide" ref="glide">
+    <div class="glide__track" data-glide-el="track">
+      <ul class="glide__slides sf-hero__slides">
+        <div v-for="(item, index) in items">
+          <!--@slot item slot for custom banner markup -->
+          <slot name="item" v-bind="{ item, index }">
```

Comment:
it's the opposite of https://github.com/DivanteLtd/storefront-ui/issues/254 i believe. According to this issue we slould provide `items`

Reply:
This slot is actually replacing a default slot that was out of the loop but didn't use `items` prop, and also didn't replace the default loop.
The default slot didn't use `items`, but the control arrows and bullets uses, and if passing `items` and using the default slot at the same time, the content is duplicated, so I think we had a bug...

Reply:
In this case I was more fixing this problem, and forgot the convention about `v-for`.

Reply:
But I think I understood the proposal, and I believe the other changes are in accordance with the convention...
I'll fix this case replacing the `<slot name="item">` to `<slot name="items">` out of the loop.

Reply:
done, I will look for more components to fix

Reply:
Awesome, thanks man!